tools/SetupFlow/DevHome.SetupFlow/Views/SetupTargetView.xaml

```diff
@@ -145,13 +145,13 @@
                         VerticalAlignment="Center">
                         <TextBlock
                             Text="{x:Bind DisplayName, Mode=OneWay}"
-                            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                            Foreground="{ThemeResource TextFillColorPrimaryBrush}"
                             Margin=" 0 0 0 0"

```

Comment:
This one was originally a secondary color but I'm changing it to be primary like the textblock next to it. They should both have actually been the same color. 

This line represents the "Microsoft Dev Box" text in the image below:

<img width="245" alt="image" src="https://github.com/user-attachments/assets/f1bdcf47-bf90-4736-b9b6-92f3e2c4ded4">